src/components/VerifyEmail/index.tsx

```diff
@@ -27,13 +30,20 @@ export default function VerifyEmail({
           password: pw,
           profileUrl: typeof data == 'string' ? data : null,
```

Comment:
```suggestion
          profileUrl: data ?? null,
```
이건 여기서 작성한 파일이 아니지만 그냥 알아만 둬